src/components/MachineManagerProvider.tsx

```diff
@@ -96,8 +97,6 @@ export const MachineManagerProvider = ({
       setCurrentMachine,
     }
 
-    engineCommandManager.machineManager = machineManagerNext
```

Comment:
I cannot find a single reference to this global registration of this key value pair. I've deleted it.